src/background.ts

```diff
@@ -1,6 +1,43 @@
+import type { User } from "./types";
+import { createFetcher } from "./utils";
+
 let ext = chrome || browser
 let dashboardUrl = "https://accounts.atoma.cloud/dashboard"
+let authData: User | null = null;
 
 ext.browserAction.onClicked.addListener(() => {
 	window.open(dashboardUrl, "_blank")
-})
+});
+
+const createTimer = () => ext.alarms.create("user-auth-refresh", { delayInMinutes: 1 })
```

Comment:
So this runs every minute or so. I think it would be good to be more conservative and only update as much as necessary